massgen/mcp_tools/native_hook_adapters/copilot_adapter.py

```
@@ -0,0 +1,282 @@
+            tool_result = input_data.get("toolResult")
+            if tool_result is not None:
+                ctx["tool_output"] = tool_result
+
+            # Serialize tool args for MassGen hook interface
+            tool_args = input_data.get("toolArgs", {})
+            arguments_str = json.dumps(tool_args) if tool_args else "{}"
+
+            try:
+                result = await hook.execute(tool_name, arguments_str, ctx)
+                return self.convert_hook_result_to_native(result, hook_type)
+            except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)